index.html

```diff
@@ -215,5 +215,57 @@ <h1 class="skills">Skills</h1>
         <li class="five">Terminal</li>
       </ul>
     </section>
+    <section class="contact">
+      <h1>
+        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+      </h1>
+      <form action="https://formspree.io/f/mbjedrdb" method="post" class="form-1">
+        <input type="text" name="Full name"
+        placeholder="Full name" id="name" 
```

Comment:
- [ ] Also as suggested by the previous reviewer, kindly make slight adjustments on the bottom border, the white space between the text and the input is large compared to the Figma design. Kindly use the [Template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) to check the approved border margin. good job  so far 👏 :

# View 
![image](https://user-images.githubusercontent.com/50990925/214769053-183522fe-955a-4b62-95ce-645515785e4c.png)

# Figma:
![image](https://user-images.githubusercontent.com/50990925/214769305-11532c74-b62f-4adf-9f08-654a92e7ad41.png)

Reply:
Hey, I see the margin-bottom of h1 text is precise as per figma template. also if I check in my side the button is also looks good I have no idea why this messed up in your side

Reply:
please advise me further

Reply:
![xvcvbv](https://user-images.githubusercontent.com/116716918/214774557-a5d7f693-a3ea-411a-886b-e09816e12290.png)

Reply:
![bvvcbfd](https://user-images.githubusercontent.com/116716918/214774747-f8523adc-e7bb-489e-8b09-cc256ec43baa.png)

And the margin-bottom is the same as per Figma

Reply:
Sorry about the late reply, I can see it's approved below already. Good job 👏